phi/playground/router.py

```diff
@@ -410,39 +483,86 @@ async def process_image(file: UploadFile) -> List[Union[str, Dict]]:
         return [encoded, image_info]
 
     @playground_router.post("/agent/run")
-    async def agent_run(body: AgentRunRequest):
-        logger.debug(f"AgentRunRequest: {body}")
-        agent = get_agent_by_id(body.agent_id, agents)
+    async def agent_run(
```

Comment:
Why do we have async and sync routes? We should only need one?

Reply:
This allows the users to choose the set of endpoints they want. If you remember the `use_async` param we use for the models that don't support async right now.